enterprise-viewer/latest/support/index.md

```diff
@@ -2,10 +2,11 @@
 title: Supported platforms
 ---
 
-The following are the supported platforms for the Alfresco Enterprise Viewer 3.5:
+The following are the supported platforms for the Alfresco Enterprise Viewer 3.5.1:
```

Comment:
Let's keep the version as either `3.5` or `3.5.x` to cover the released versions, `3.5.0` and `3.5.1`.
```suggestion
The following are the supported platforms for the Alfresco Enterprise Viewer 3.5:
```